.github/workflows/scripts/is-community-contributor.js

```diff
@@ -1,92 +1,49 @@
-const getUserFromContext = ( types, context ) => {
-	for ( const type of types ) {
-		if ( context.payload[ type ] && context.payload[ type ].user && context.payload[ type ].user.login ) {
-			return context.payload[ type ].user;
-		}
-	}
-	return false;
-};
-
-const userIsBot = user => user.type && user.type === 'Bot';
-
-const userInOrg = async ( github, username, org ) => {
-	console.log( 'Checking for user %s in org %s', username, org );
-	try {
-		// First attempt to check memberships.
-		const result = await github.request( 'GET /orgs/{org}/memberships/{username}', {
-			org,
+// Note you'll need to install this dependency as part of your workflow.
+const { Octokit } = require('@octokit/action');
+
+// Note that this script assumes you set GITHUB_TOKEN in env, if you don't
+// this won't work.
+const octokit = new Octokit();
+
+const getPRAuthor = (payload) => {
+	return payload?.pull_request?.user?.login || null;
+}
```

Comment:
```suggestion
const getIssueAuthor = (payload) => {
	return payload?.issue?.user?.login || payload?.pull_request?.user?.login || null;
}
```

The previous functionality didn't work with issues. I updated this and renamed the function (since each PR is an issue, but not vise-versa).